portal-bridge/src/constants.rs

```diff
@@ -18,9 +18,16 @@ pub const BEACON_GENESIS_TIME: u64 = 1606824023;
 // overloaded and not performing well
 pub const HTTP_REQUEST_TIMEOUT: Duration = Duration::from_secs(20);
 
-// This is the maximum number of active blocks being gossiped
+// The maximum number of active blocks being gossiped. Note that this doesn't
```

Comment:
nit: should we say "blocks" or "content key/value" or something similar (as I believe blocks is only correct in history network context).

Reply:
I think it's fine. At the bottom of this comment, I'm explicit about `Currently, this is only used in the history & era1 bridges`, which are both in the history network context, in which "blocks" is actually the correct unit that the semaphore is limiting. Aka, each block being limited contains multiple key/value pair requests